lib/util/github/graphql/cache-strategies/abstract-cache-strategy.ts

```diff
@@ -101,7 +101,6 @@ export abstract class AbstractGithubGraphqlCacheStrategy<
       const oldItem = cachedItems[version];
       if (oldItem && this.isStabilized(oldItem)) {
         isPaginationDone = true;
```

Comment:
So it will always mark pagination done as long as one item is stabilized?

Reply:
Yes, and we won't fetch the second page if any stabilized item was encountered in the first one.